sklearn/linear_model/ridge.py

```diff
@@ -526,6 +526,8 @@ class RidgeClassifier(LinearClassifierMixin, _BaseRidge):
 
     normalize : boolean, optional, default False
         If True, the regressors X will be normalized before regression.
+        Note that regressors are always centered no matter what and no
```

Comment:
I don't understand (maybe because it is out of context) what "no transformation is required for prediction" means. No transformation on X?